client/push/push_manager.js

```diff
@@ -25,13 +25,40 @@ export default class Push {
   }
 
   /**
+   *  get push permission status
+   */
+  getPushPermissionStatus () {
+    // query push permissions
+    return new Promise(function(resolve, reject) {
+      navigator.permissions.query({name:'push', userVisibleOnly:true})
+      .then(function(permissionStatus) {
+        console.log('push permission state is ', permissionStatus.state);
+        resolve(permissionStatus.state);
+      });
+    });
+  }
+
+  /**
+   * get push subscription with uuid from server
+   */
+  getPushSubscription() {
+    let pushSubscrip;
+    self.registration.pushManager.getSubscription().then(subscription => {
```

Comment:
If we can use '=>' arrow function , how about to apply all callback to this file for consistency?